packages/connect-conformance/src/callback-client.ts

```diff
@@ -184,6 +221,7 @@ async function serverStream(
       },
       {
         headers: reqHeader,
+        signal: controller.signal,
```

Comment:
It's preferable to cancel with the cancel function that the callback client returns - it's the API we want to test here. Can we continue to use it?

Reply:
If we use `cancel` we can't check for signal.aborted

Reply:
Tracking it ourselves is straight-forward: https://github.com/connectrpc/connect-es/pull/1211